packages/client-sdk-nodejs/src/config/transport/transport-strategy.ts

```diff
@@ -94,6 +95,15 @@ export class StaticGrpcConfiguration implements GrpcConfiguration {
       // This is the previously hardcoded value and a safe default for most environments.
       this.numClients = 6;
     }
+    if (
+      props.concurrentRequestsLimit !== undefined &&
+      props.concurrentRequestsLimit !== null
+    ) {
+      this.concurrentRequestsLimit = props.concurrentRequestsLimit;
+    } else {
+      // This is a hardcoded limit that can be tuned, but it likely a safe default for most environments.
+      this.concurrentRequestsLimit = 100;
```

Comment:
We should externalize this to a global const and document that a default is applied if the user doesn't set it.